tools/ZeroC.Ice.Slice.Tools.CSharp/README.md

```diff
@@ -0,0 +1,20 @@
+# Slice Tools for Ice for C#
```

Comment:
I don't like the double for. What about simply: Slice Tools for Ice C#.

Then, I think we should use the same name consistently to avoid confusion. I would use "Slice tools for Ice C#" and "Slice tools for Ice C++" and drop the builder terminology for these tools/packages.